### DISCUSSION TITLE
Tools wording unclear?

### DISCUSSION THREAD

**Discussion by PineappleRind** [spacedriveapp/spacedrive#926] [General]
When I saw the Duplicate Finder tool I thought it was going to duplicate Finder (somehow) 😅 until I saw how the others were worded 

<img width="155" alt="image" src="https://github.com/spacedriveapp/spacedrive/assets/65303441/6a6c23ef-df6b-404e-9e8b-2e53dcb24da2">

imo having them be called "Find Duplicates", "Clean Cache", "Encode Media"... would be better

what do you all think?

**Comment by utkubakir** [comment]
I think it's fine but we will keep this in mind once we start working on them. They are just a concept atm.
